doc/src/cylc-user-guide/cug.tex

```diff
@@ -4779,6 +4779,20 @@ \subsubsection{Parameter Offsets In The Graph}
     graph = "proc_small => proc_big => proc_huge"
 \end{lstlisting}
 
+However, a mixture of conditional logic in these statements may create
+surprises. For example:
+
+\begin{lstlisting}
+    graph = foo<m-1> & baz => foo<m>  # for m = cat, dog
+\end{lstlisting}
+currently expands to:
+\begin{lstlisting}
+    graph = foo_cat & baz => foo_dog
+# when users may expect it to be:
+#    graph = foo_cat => foo_dog
+#    graph = baz => foo_cat & foo_dog
+\end{lstlisting}
+
```

Comment:
I think this should be acknowledged as a bug, which it is.  A "surprise" sounds like it could be intentional.

Reply:
OK. Done with more advice.